configure-ironic.sh

```diff
@@ -2,6 +2,14 @@
 
 . /bin/ironic-common.sh
 
+USE_HTTP_BASIC=${USE_HTTP_BASIC:-false}
+IRONIC_HTTP_BASIC_FILE=${IRONIC_HTTP_BASIC_FILE:-"/shared/htpasswd-ironic"}
+IRONIC_HTTP_BASIC_USERNAME=${IRONIC_HTTP_BASIC_USERNAME:-"change_me"}
+IRONIC_HTTP_BASIC_PASSWORD=${IRONIC_HTTP_BASIC_PASSWORD:-"change_me"}
+JSONRPC_HTTP_BASIC_FILE=${JSONRPC_HTTP_BASIC_FILE:-"/shared/htpasswd-json-rpc"}
+JSONRPC_HTTP_BASIC_USERNAME=${JSONRPC_HTTP_BASIC_USERNAME:-"change_me"}
+JSONRPC_HTTP_BASIC_PASSWORD=${JSONRPC_HTTP_BASIC_PASSWORD:-"change_me"}
```

Comment:
JSON RPC credentials are not user-visible, let's just generate them on fly.

Reply:
agree that they are not user-visible, so we just generate some random username/password to be used?